shared/fs/header/header.desktop.js

```diff
@@ -106,27 +109,36 @@ const styleFolderHeaderEnd = {
   alignItems: 'center',
   paddingLeft: 16,
   paddingRight: 16,
+  flexShrink: 0,
 }
 
 const folderBreadcrumbStyle = {
   ...globalStyles.flexBoxRow,
   alignItems: 'center',
   paddingLeft: 0,
   paddingRight: 0,
+  flexShrink: 0,
 }
 
+const lastFolderBreadcrumbStyle = {
+  ...folderBreadcrumbStyle,
+  flexShrink: 1,
+}
+
+const stylesLastNameText = {}
```

Comment:
This is where I tried `overflowWrap: 'break-word'` but it didn't work. I also tried adding a few `width: '100%'`s and it didn't help either.